src/deployments.js

```diff
@@ -1,18 +1,23 @@
 const { Octokit } = require('@octokit/rest');
 const { graphql } = require('@octokit/graphql');
 const WORKFLOW_DEPLOY = 'workflowdeploy';
-const ALLOWED_STATUSES = [
-  'success',
-  'error',
-  'failure',
-  'inactive',
-  'in_progress',
-  'queued',
-  'pending'
-];
+const ALLOWED_STATUSES = {
+  SUCCESS: 'success',
+  ERROR: 'error',
+  FAILURE: 'failure',
+  INACTIVE: 'inactive',
+  IN_PROGRESS: 'in_progress',
+  QUEUED: 'queued',
+  PENDING: 'pending'
+};
+
+const createOctokitClient = token => new Octokit({ auth: token });
+const createOctokitGraphQLClient = token =>
+  graphql.defaults({ headers: { authorization: `token ${token}` } });
```

Comment:
Is there an advantage to defining a function that creates an octokit client as opposed to creating a single client that is passed to the functions that need it?  It seems like having a single client that all the different functions use might be better than each function creating its own version of the client.

Reply:
This funciton is getting used by the deployment testing file, too.  I wanted to make sure the setting params for these clients were the same in both the runtime code as well as the jest test.